### PR TITLE
[zh] sync configure-persistent-volume-storage kubeadm-certs feature-gates-removed working-with-objects/_index.md

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -286,26 +286,29 @@ to keep them both in sync.
 {{< /note >}}
 
 <!--
-`kubeadm certs renew` provides the following options:
+`kubeadm certs renew` can renew any specific certificate or, with the subcommand `all`, it can renew all of them, as shown below:
 -->
-`kubeadm certs renew` 提供以下选项：
+`kubeadm certs renew` 可以更新任何特定的证书，或者使用子命令 `all`，
+它可以更新所有的证书，如下所示：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:292]:
```suggestion
`kubeadm certs renew` 可以更新任何特定的证书，或者使用子命令 `all`
更新所有的证书，如下所示：
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 2]
```diff
@@ -286,26 +286,29 @@ to keep them both in sync.
...
-  signing requests (without actually renewing certificates in place); see next paragraph for more
-  information.
-
-- It's also possible to renew a single certificate instead of all.
+Clusters built with kubeadm often copy the `admin.conf` certificate into `$HOME/.kube/config`, as instructed in [Creating a cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/). On such a system, to update the contents of `$HOME/.kube/config` after renewing the `admin.conf` you must run the following commands:
 -->
-- `--csr-only` 可用于经过一个外部 CA 生成的证书签名请求来更新证书（无需实际替换更新证书）；
-  更多信息请参见下节。
+使用 kubeadm 构建的集群通常会将 `admin.conf` 证书复制到 `$HOME/.kube/config` 中，
+如[使用 kubeadm 创建集群](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)中所指示的那样。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:303]:
```suggestion
如[使用 kubeadm 创建集群](/zh-cn/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)中所指示的那样。
```

**Review by windsonsea — COMMENTED** [review]
/approve

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**